cpp/include/cudf/round.hpp

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cudf/column/column.hpp>
+
+namespace cudf {
+
+/**
+ * @addtogroup transformation_unaryops
+ * @{
+ * @file
+ * @brief Column APIs for round
+ */
+
+/**
+ * @brief Different rounding methods for `cudf::round`
+ */
+enum class rounding_method : int32_t { HALF_UP };
+
+/**
+ * @brief Rounds all the values in a column to the the nearest multiple of `10^decimal_places`.
```

Comment:
Presumably this will change after the HALF_EVEN and other PR...

Reply:
yep, i started working on the HALF_EVEN in a branch dependent on this one